### PR TITLE
Fix redesigned CLI commands

### PR DESCRIPTION
**Pull request by dtobolik**
Expand CLI aliases to full commands in documentation, tests, comments, CLI outputs

Also regenarated workflow png to contain the full commands

### CODE REVIEW

File: containers/rocm/README.md [hunk 1]
```diff
@@ -68,4 +68,4 @@ make rocm-gfx1100 BUILD_ARGS=
 
 ## Known issues
 
-AMD Instinct MI210 with ISA `amdgcn-amd-amdhsa--gfx90a:sramecc+:xnack-` is not supported by Fedora build `rocblas-6.0.2-3`. As of late April 2024, Fedora has `gfx90a:xnack+` and `gfx90a:xnack-` but lacks `gfx90a:sramecc+:xnack-`.
\ No newline at end of file
+AMD Instinct MI210 with ISA `amdgcn-amd-amdhsa--gfx90a:sramecc+:xnack-` is not supported by Fedora build `rocblas-6.0.2-3`. As of late April 2024, Fedora has `gfx90a:xnack+` and `gfx90a:xnack-` but lacks `gfx90a:sramecc+:xnack-`.
```
Comment (tiran) [inline containers/rocm/README.md:71]: What did you change in this line?

**Comment by tiran** [comment]
Thank you! The PR looks okay. I just wonder why there is one line marked as changed. Did you add or remove a newline?

File: containers/rocm/README.md (hunk 1 above)
Comment (dtobolik) [reply containers/rocm/README.md]: Sorry, I added newline. Fixed now

**Review by tiran — APPROVED** [review]
Awesome! Thank you for your contributions. The changes look okay to me.

**Review by bjhargrave — APPROVED** [review]

**Pull request merged**